src/redux/wallets/actions/util-actions.ts

```diff
@@ -122,11 +122,12 @@ export const signAndSendTransactions = (specificIndex?: number) => async (
                         // found a tx with the same nonce
                         const txStatus = await client.utils.getTransactionStatus(outTx.id, {
                             broadcastedOnBlock: outTx.broadcastedOnBlock,
-                            currentBlockNumber
+                            currentBlockNumber,
+                            address: account.address
                         });
 
                         // if the status oif the tx is DROPPED we can reuse the nonce
-                        if (txStatus === TransactionStatus.DROPPED) {
+                        if (txStatus === TransactionStatus.DROPPED || !txStatus) {
```

Comment:
@krisboit please confirm that this won't affect ZIL flow

Reply:
`getTransactionStatus` should always return a value

Reply:
check again